packages/sparqlee/README.md

```diff
@@ -249,114 +236,6 @@ this at least allows evaluation with multiple bindings to have the same `now` va
 To be fully spec compliant, the IRI/URI functions should take into account base IRI of the query,
 which you can provide as `baseIRI: string` to the config.
 
-## Spec compliance
```

Comment:
I decided to remove this section. I think we should be completely (or at least as far as we think?) spec compliant? If not, issues would have been exposed after all these years. Additionally, the table hasn't been updated for very long and I think it does not add any value anymore.